core-ui/src/components/Extensibility/jsonataWrapper.js

```diff
@@ -0,0 +1,18 @@
+import jsonata from 'jsonata';
+import { isEqual } from 'lodash';
+
+export function jsonataWrapper(expression) {
+  const exp = jsonata(expression);
+
+  exp.registerFunction('selectorMatchByLabels', (pod, labels) => {
+    if (!pod.metadata?.labels || !labels) return false;
+
+    const podLabels = Object?.entries(pod.metadata?.labels);
+    const resourceLabels = Object?.entries(labels);
```

Comment:
`Object` is always defined in JS engine, no need for `?.` operator. Besides, `pod.metadata.labels` will always be defined here (as checked in line 8).

```suggestion
    const podLabels = Object.entries(pod.metadata.labels);
    const resourceLabels = Object.entries(labels);
```